my-app/src/App.css

```diff
@@ -57,4 +57,11 @@
 
 a{
   color:white
+}
+
+button:disabled,
+button[disabled]{
+  border: 1px solid #999999;
+  background-color: #cccccc;
```

Comment:
If this color(#cccccc) has been used only once then it's fine otherwise you could have created a variable for it.